sentry/src/main/java/io/sentry/protocol/MeasurementValue.java

```diff
@@ -0,0 +1,13 @@
+package io.sentry.protocol;
+
+import org.jetbrains.annotations.ApiStatus;
+
+@ApiStatus.Internal
+public final class MeasurementValue {
```

Comment:
I wonder if we're better off without this class and using the `float` directly to avoid allocating these objects

Reply:
could be easily removed, its not a public API, but the idea was to be extendable like the API is, its an object with a single float property